readme.md

```diff
@@ -64,53 +64,54 @@ docker run -e TEST_NAME=all -it --rm databench
 
 | Language       | Time (5k posts)                       | 20k posts                              | 60k posts                           | Total     |
 | -------------- | ------------------------------------- | -------------------------------------- | ----------------------------------- | --------- |
-| _Julia HO_[^1] | 8.15 ms | 54.33 ms | 143.67 ms | 206.15 ms |
-| D | 25.58 ms | $\textsf{\color{lightgreen}314.59 ms}$ | $\textsf{\color{lightgreen}2.70 s}$ | 3.04 s |
-| C++ | 23.15 ms | 322.33 ms | 2.79 s | 3.13 s |
-| Nim | $\textsf{\color{lightgreen}22.10 ms}$ | 318.06 ms | 2.84 s | 3.18 s |
-| Go | 22.72 ms | 321.73 ms | 2.85 s | 3.20 s |
-| Zig | 23.00 ms | 339.67 ms | 3.01 s | 3.38 s |
-| Rust | 23.59 ms | 340.75 ms | 3.04 s | 3.40 s |
-| C# (AOT) | 26.50 ms | 371.96 ms | 3.25 s | 3.65 s |
-| C# (JIT) | 27.49 ms | 375.33 ms | 3.28 s | 3.68 s |
-| F# (AOT) | 27.23 ms | 390.33 ms | 3.41 s | 3.83 s |
-| Julia | 27.15 ms | 364.00 ms | 4.22 s | 4.61 s |
-| F# (JIT) | 91.46 ms | 529.00 ms | 4.04 s | 4.66 s |
-| Odin | 48.22 ms | 574.75 ms | 5.05 s | 5.67 s |
-| Java (GraalVM) | 50.00 ms | 745.33 ms | 5.55 s | 6.34 s |
-| Swift | 51.95 ms | 716.99 ms | 6.20 s | 6.97 s |
-| Vlang | 49.01 ms | 742.33 ms | 6.59 s | 7.38 s |
-| Crystal | 60.37 ms | 914.35 ms | 7.99 s | 8.97 s |
-| LuaJIT | 99.83 ms | 1.17 s | 12.00 s | 13.27 s |
-| JS (Bun) | 122.62 ms | 1.49 s | 13.14 s | 14.75 s |
-| JS (Node) | 139.00 ms | 2.29 s | 15.77 s | 18.20 s |
-| JS (Deno) | 165.15 ms | 1.88 s | 17.23 s | 19.28 s |
+| _Julia HO_[^1] | 8.15 ms | 54.00 ms | 138.00 ms | 200.15 ms |
+| D | 24.73 ms | 315.66 ms | 2.70 s | 3.04 s |
+| Nim | 22.15 ms | 317.63 ms | 2.84 s | 3.18 s |
+| Go | 22.69 ms | 322.86 ms | 2.86 s | 3.21 s |
+| Zig | 23.00 ms | 337.00 ms | 2.99 s | 3.35 s |
+| Rust | 23.53 ms | 340.58 ms | 3.03 s | 3.40 s |
+| C# (AOT) | 26.34 ms | 373.28 ms | 3.26 s | 3.65 s |
+| C# (JIT) | 27.52 ms | 376.84 ms | 3.26 s | 3.67 s |
+| F# (AOT) | 27.38 ms | 392.00 ms | 3.42 s | 3.84 s |
+| F# (JIT) | 91.46 ms | 532.33 ms | 4.04 s | 4.67 s |
```

Comment:
What happened to the first time here... Is vCPU really this unstable

Reply:
Are you referencing f#? It has consistently gotten that score over the last ~10 runs so I consider that stable. The jit just performs better on longer runs.